examples/dataform_json_checks/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//tests:__subpackages__"])
```

Comment:
You are right about this particular `BUILD` file. It can be deleted. *BUT* all other `BUILD` files have to remain because each `dataform.json` test requires a new independent folder.